gitgud/skills/test_util.py

```diff
@@ -28,3 +28,32 @@ def test_goal():
     for skill in all_skills:
         for level in skill:
             os.path.isfile(level.goal_path)
+
+
+def test_init_NL():
+    nltest = NamedList(['foo', 'bar'], [511, 522])
+
+
+def test_getitem_NL():
+    nltest = NamedList(['foo', 'bar', 'baz'], [5, 7, 9])
+    assert nltest[0] == 5
+    assert nltest['baz'] == 9
+
+
+def test_iter_NL():
+    ['' for skill in all_skills]
+
+
+def test_len_NL():
+    assert len(all_skills[0]) == 4
```

Comment:
If we add a level, we have to change this, but that might get annoying. Can you change this to use `nltest` like you do in the other methods